src/helm/helmfile.yaml

```diff
@@ -26,6 +26,7 @@ releases:
     values:
       - auth:
           password: pass
+        architecture: standalone
```

Comment:
Do you know why we add this line?

Reply:
https://github.com/bitnami/charts/blob/main/bitnami/redis/README.md#standalone

Reply:
https://github.com/bitnami/charts/blob/main/bitnami/redis/README.md#standalone